proto/one/operations/sample/enum_import_status.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+// List of potential statuses for an analyte input
+enum EnumImportStatus {
```

Comment:
In order to avoid having to maintain and exhaustive list of potential errors can we have one that is a generic error to use when we don't have an exact error?  What happens if multiple fields are missing from the file?

Another potential option here would be to use the existing ApiError class instead of this new Enum.  That would give us a lot more flexibility and it seems like there is some overlap with existing SharedErrors.  The downside is it might be harder to parse and react to.

Reply:
For the first portion, would IMPORTSTATUS_UNKNOWN not account for this? If not, what would you like it be named?

As for the second comment, this enum was part of the suggested protobuf and some of the errors are specific, so I would like to keep it here so that its easier to parse out descriptive errors like you mentioned.

Reply:
It might be a little bit repetitive, but I could potentially see having a error_unknown option added.  It's a little different than unknown = 0.

Thinking about the interactions between the Processor and API, I'd probably lean towards having errors in protobuf for this chunk of work.  I think having more well defined errors and logic around them is probably what Steve was advocating for.

Reply:
Gotcha, I will add an IMPORTSTATUS_ERROR_UNKNOWN as well.

Reply:
I think ERROR_UNKNOWN is a good addition.  Still curious what should happen if the file is missing multiple required fields like sampleNumber and AnalyteName

Reply:
After talking more with Dan, we think that a more generic import status enum combined with the errorDetails field on the analyte result protobuf would be a cleaner way to handle all potential issues.